src/components/SecondaryPanes/EventListeners.css

```diff
@@ -33,7 +60,14 @@ html[dir="rtl"] .event-listeners-content .arrow.expanded {
 	margin-inline-start: 30px;
 }
 
+.event-listener-event span {
```

Comment:
This selector could impact too many <span> elements, especially if we add some in the future.
Maybe `.event-listener-event label span`, or a new class like `.event-listener-name`.